nixos/modules/security/please.nix

```diff
@@ -0,0 +1,84 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.security.please;
+in
+{
+  options.security.please = {
+    enable = mkEnableOption (mdDoc ''
+      please, a tool which allows non-root users to execute commands as root
+    '');
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.please;
+      defaultText = literalExpression "pkgs.please";
+      description = mdDoc ''
+        Which package to use for {command}`please`;
+      '';
+    };
+
+    settings = mkOption {
+      type = types.attrs;
+      default = { };
+      example = {
+        jim_become_root = {
+          name = "jim";
+          target = "root";
+          rule = ".*";
+          require_pass = false;
+        };
+        jim_hosts = {
+          name = "jim";
+          type = "edit";
+          target = "root";
+          rule = "/etc/hosts";
+          editmode = 644;
+          require_pass = false;
+        };
+      };
+      description = mdDoc ''
+        Please configuration. Refer to
+        <https://github.com/edneville/please/blob/master/please.ini.md> for
+        details.
+      '';
+    };
+  };
+
+  config = mkIf cfg.enable {
+    security.wrappers =
+      let
+        owner = "root";
+        group = "root";
+        setuid = true;
+      in
+      {
+        please = {
+          source = "${cfg.package.out}/bin/please";
+          inherit owner group setuid;
+        };
+        pleaseedit = {
+          source = "${cfg.package.out}/bin/pleaseedit";
```

Comment:
Nit:

These shouldn't need to refer to the `out` output by name, since it appears there is only 1 output at the moment. If you want to be "more correct" in case the binaries get moved to the `bin` output, you can use `lib.getBin cfg.package`. I personally would just drop the `.out` from both and call it good, though. Up to you.